packages/cardano-services/src/util/polling/types.ts

```diff
@@ -0,0 +1,6 @@
+export type Disposer = () => void;
+
+export interface EpochMonitor {
+  onEpochRollover(callback: () => void): Disposer;
+  getEpoch(): Promise<number>;
```

Comment:
Maybe let's remove this from the interface (keeping it on DbSyncEpochPollService), because it's quite confusing as it's implemented now. I would expect this to return current epoch (make a query) instead of return last known cached epoch.

```suggestion
```

Reply:
No, I don't think it should send an actual query, as that will affect the tests. I think it just needs to be renamed as per [comment below](https://github.com/input-output-hk/cardano-js-sdk/pull/370#discussion_r942383745)